pkg/api/client.go

```diff
@@ -97,3 +100,28 @@ func FromConfig(conf *config.Config) Client {
 	}
 
 }
+
+func GetErrorResponse(err error, methodName string) error {
+	if err == nil {
+		return nil
+	}
+	errResponse := &rawclient.ErrorResponse{}
+	newErr := json.Unmarshal([]byte(err.Error()), errResponse)
+	if newErr != nil {
+		return err
+	}
+
+	errList := errors.New(methodName)
+	if errResponse.GqlErrors != nil {
+		for _, err := range *errResponse.GqlErrors {
```

Comment:
We should make a reduce function in pluralsh/polly to eliminate these common handwritten loops